src/session/sessions.ts

```diff
@@ -0,0 +1,249 @@
+//#region > Imports
+//> Session
+// Contains the base session
+import Session from "./index";
+//> Templates
+// Contains the main template for the sessions
+import { IMainTemplate } from "../templates/index";
+// Contains the snek template
+import SnekTemplate from "../templates/snek/index";
+//> Cookie Utils
+// Contains tools for cookie handeling
+import {
+  cookieChecker,
+  getCookie,
+  setCookie,
+  deleteCookie,
+} from "./cookie-utils";
+//> Tasks
+// Contains snek tasks
+import SnekTasks from "../templates/snek/gql/tasks/index";
+//> Interfaces
+// Contains the interface for gql queries, mutations and subscriptions
+import { DocumentNode } from "graphql";
+// Contains the interface for the apollo endpoint
+import { ApolloEndpoint } from "../../src/endpoints/index";
+// Contains basic session interfaces
+import { IAuth, User, UserData } from "./index";
+//#endregion
+
+/** @class A Github SubSession  */
+class GithubSession extends Session {
+  /**
+   * Creates an instance of a GithubSession.
+   *
+   * @constructor
+   * @param {string} sId A session name
+   * @param {Endpoint} ep A endpoint
+   * @param {IMainTemplate} template A template set
+   */
+  constructor(
+    sId: string,
+    public ep: ApolloEndpoint,
+    public template: IMainTemplate
```

Comment:
Missing comma.

Reply:
In this case a comma is not needed because this is not an object declaration over multiple lines. This are constructor arguments.